test/e2e/e2e-test-job.yaml.tmpl

```diff
@@ -17,6 +17,7 @@ metadata:
   name: $JOB_NAME
   namespace: e2e-test
 spec:
+  backoffLimit: 0
```

Comment:
the default limit is 6 retries. I could see maybe bumping it to 1 to get 1 automatic retry on a failed integration test, but largely failures seem to be real failures (and can be retried using `/retest`) so set to 0 initially

Reply:
This sounds good to me.